src/section_handlers/document_tracking.py

```diff
@@ -6,6 +6,7 @@
 from src.common.common import SectionHandler
 from utils import get_utc_timestamp
 from operator import itemgetter
+from contextlib import suppress
```

Comment:
Nit: Where is this used (browser finds only run occurrence in change set (the import))?